.travis.yml

```diff
@@ -59,6 +63,7 @@ matrix:
       env: TARGET=go
 
 before_install:
+  - ./.travis/before-install-mvn3.3.9-trusty.sh
```

Comment:
Looks like @lecode-official is on holiday again? I've made second PR to fix this issue.